lib/events/filesessions/filestream.go

```diff
@@ -142,7 +142,7 @@ func (h *Handler) CompleteUpload(ctx context.Context, upload events.StreamUpload
 		partPath := h.partPath(upload, part.Number)
 		file, err := os.Open(partPath)
 		if err != nil {
-			return trace.ConvertSystemError(err)
+			return trace.Wrap(err, "failed to open part file for upload")
```

Comment:
`ConvertSystemError` does the wrong conversion for path error - the error message it provides is [confusing](https://github.com/gravitational/trace/blob/2897205b8d7fb7a00817fdc2e5a859e1b296eff9/errors.go#L289) and the constructor it chooses (`AccessDeniedError`) is also invalid.